tests/test_files.py

```diff
@@ -1182,43 +1182,60 @@ def test_different_images():
 
 
 @pytest.fixture
-def graphie_filepaths(tmpdir):
-    """Create temporary SVG and JSON files for testing"""
-    # Create base filepath without extension
-    base_path = os.path.join(str(tmpdir), "testgraphie")
-
-    # Write SVG file
-    with open(f"{base_path}.svg", "w") as f:
-        f.write(SAMPLE_SVG)
-
-    # Write JSON file
-    with open(f"{base_path}-data.json", "w") as f:
-        f.write(SAMPLE_JSON)
+def mock_download_session():
+    """Mock the download session to return appropriate content for SVG and JSON requests"""
+    with patch("ricecooker.config.DOWNLOAD_SESSION") as mock_session:
+        mock_response = MagicMock()
+        mock_response.headers = {}
+
+        def iter_content(chunk_size=None):
+            content = None
+            if mock_response.url.endswith(".svg"):
+                content = SAMPLE_SVG.encode("utf-8")
+            elif mock_response.url.endswith("-data.json"):
+                content = SAMPLE_JSON.encode("utf-8")
+            else:
+                raise Exception(f"Unexpected URL pattern: {mock_response.url}")
+
+            if chunk_size is None:
+                yield content
+                return
+
+            # Yield content in chunks of specified size
+            for i in range(0, len(content), chunk_size):
```

Comment:
I don't understand which scenarios we would actually be passing in a (test?) chunk_size here. I think this is just in different files? i.e. `chefs.py` https://github.com/marcellamaki/ricecooker/blob/develop/ricecooker/chefs.py#L859

Reply:
The content is always going to be smaller than the chunk size - I just wanted to make sure we were iterating over it, just in case we wanted to make it bigger at some point.